src/mumble/GlobalShortcut_xdp.cpp

```diff
@@ -0,0 +1,189 @@
+// Copyright 2007-2022 Aleix Pol Gonzalez <aleixpol@kde.org>.
+// All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
```

Comment:
The copyright date is wrong.

Also, we would strongly prefer not having individual copyright notes from everyone that eventually makes changes to this (or any other) file. The individual contributors are tracked via git anyway.
Now, we can't (and won't try to) force you to remove your copyright statement, but as I said: if you are okay with that, we'd definitely prefer it if you removed it and left only the general copyright notice.